daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -180,27 +180,6 @@ class DFDLTestSuite private[tdml] (
   val shouldDoWarningComparisonOnCrossTests: Boolean)
   extends HasSetDebugger {
 
-  // Uncomment to force conversion of all test suites to use Runner(...) instead.
-  // That avoids creating the test suites repeatedly, but also leaks memory unless
-  // you have an @AfterClass shutdown method in the object that calls runner.reset() at end.
-  @deprecated("Use Runner(...) instead.", "3.2.0")
```

Comment:
Removing this might break existing tests that don't live in the Daffodil repo. It was only deprecated a couple releases ago. We might want to give more time to fix things unless this caused an issue.